.claude/skills/git/SKILL.md

```
@@ -0,0 +1,26 @@
+---
+name: git
+description: Esposter git workflow conventions — commit message format, safety rules, and branch hygiene. Apply when running git operations or advising on source control workflows.
+---
+
+# Git Conventions
+
+## Commit Message Format
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)